website/src/components/TaskPage/TaskPage.tsx

```diff
@@ -0,0 +1,40 @@
+import Head from "next/head";
+import { useTranslation } from "next-i18next";
+import { TaskEmptyState } from "src/components/EmptyState";
+import { LoadingScreen } from "src/components/Loading/LoadingScreen";
+import { Task } from "src/components/Tasks/Task";
+import { TaskInfos } from "src/components/Tasks/TaskTypes";
+import { apiHooksByType, ERROR_CODES } from "src/lib/constants";
+import { getTypeSafei18nKey } from "src/lib/i18n";
+import { TaskType } from "src/types/Task";
+
+type TaskPageProps = {
+  type: TaskType;
+};
+
+export const TaskPage = ({ type }: TaskPageProps) => {
+  const { t } = useTranslation(["tasks", "common"]);
+  const apiHook = apiHooksByType[type];
+  const { tasks, isLoading, reset, trigger, error } = apiHook(type);
+  const taskType = TaskInfos.find((taskType) => taskType.type === type);
```

Comment:
could you please rename this variable to `taskInfo`?

Reply:
Sure, done.